### ISSUE TITLE
[ter-indent] not work with some comments

### ISSUE THREAD

**Issue by ekoneko** [buzinas/tslint-eslint-rules#275]
➜  tslint-eslint-rules git:(master) ✗ npx tslint --version
5.7.0
➜  tslint-eslint-rules git:(master) ✗ npx tsc --version
Version 2.5.3

Run this test case:

````js
ruleTester.addTestGroup('indent-number', 'should force a certain indentation number', [
  {
    code: dedent`
    /**
     * test
     * @param {Object | Immutable.Record(File)} file
     * @param {String} type
     */
    export const test = (file: any = {}, type) => {
      //
    }`,
    options: [2]
  }
])
````

The result is:

```
ter-indent: Expected indentation of NaN spaces but found 0.
```

When we remove the function comments, this test works fine.
